.travis.yml

```diff
@@ -9,7 +9,7 @@ env:
   - BABEL_VERSION="6"
   - BABEL_VERSION="7.0.0-beta.0"
 before_install:
-  - npm install babel-cli@^"$BABEL_VERSION" babel-core@^"$BABEL_VERSION" babel-generator@^"$BABEL_VERSION" babel-plugin-syntax-async-functions@^"$BABEL_VERSION" babel-plugin-transform-async-to-generator@^"$BABEL_VERSION" babel-plugin-transform-es2015-destructuring@^"$BABEL_VERSION" babel-plugin-transform-es2015-modules-commonjs@^"$BABEL_VERSION" babel-plugin-transform-flow-strip-types@^"$BABEL_VERSION" babel-plugin-transform-regenerator@^"$BABEL_VERSION" babel-preset-es2015@^"$BABEL_VERSION" babel-preset-latest@^"$BABEL_VERSION" babel-register@^"$BABEL_VERSION"
+  - npm install babel-cli@^"$BABEL_VERSION" babel-core@^"$BABEL_VERSION" babel-generator@^"$BABEL_VERSION" babel-plugin-syntax-async-functions@^"$BABEL_VERSION" babel-plugin-transform-async-to-generator@^"$BABEL_VERSION" babel-plugin-transform-es2015-destructuring@^"$BABEL_VERSION" babel-plugin-transform-es2015-modules-commonjs@^"$BABEL_VERSION" babel-plugin-transform-flow-strip-types@^"$BABEL_VERSION" babel-plugin-transform-regenerator@^"$BABEL_VERSION" babel-preset-es2015@^"$BABEL_VERSION" babel-preset-env@^"$BABEL_VERSION" babel-register@^"$BABEL_VERSION"
```

Comment:
N.B. that `babel-preset-latest` is deprecated in favor of [`babel-preset-env`](https://babeljs.io/docs/plugins/preset-env/): "Without any configuration options, babel-preset-env behaves exactly the same as babel-preset-latest (or babel-preset-es2015, babel-preset-es2016, and babel-preset-es2017 together)."